packages/api/src/users/users.controller.ts

```diff
@@ -1,15 +1,15 @@
 import { Controller, Get } from '@nestjs/common';
 
-import { GetAllUsersResponse } from '@coderscamp/shared/models/user';
-
-import { UsersService } from './users.service';
+import { UsersMapper } from './users.mapper';
+import type { UserDTO } from './users.model';
+import { UsersRepository } from './users.repository';
 
 @Controller('users')
 export class UsersController {
-  constructor(private readonly usersService: UsersService) {}
+  constructor(private readonly usersRepository: UsersRepository) {}
 
   @Get('/')
-  async getAll(): Promise<GetAllUsersResponse> {
-    return this.usersService.getAll();
+  async getAll(): Promise<UserDTO[]> {
```

Comment:
The return type of the controller should be placed in the shared package (like GetAllUsersResponse was before) so we can share it with the frontend.